keras_nlp/models/bert/bert_tasks.py

```diff
@@ -15,16 +15,15 @@
 
 from tensorflow import keras
 
-from keras_nlp.models.bert.bert_checkpoints import checkpoints
+from keras_nlp.models.bert.bert_models import Bert
 from keras_nlp.models.bert.bert_models import bert_kernel_initializer
-from keras_nlp.models.bert.bert_models import model_class_by_name
+from keras_nlp.models.bert.bert_presets import backbone_presets
```

Comment:
Should this be exposed as a class property?

Reply:
(and since it's on the `Bert` class it should presumably be called `presets`. Though one could image it called `backbone_presets` on a `BertClassifier`, to leave the possibility of `presets` for end-to-end classifier weights, etc.)

Reply:
Referenced `Bert.presets` directly. We can save `BertClassifier.presets` for task-specific checkpoints.